src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -540,6 +540,8 @@ def load_arguments(self, _):
         c.argument('display_name', arg_type=ts_display_name_type)
         c.argument('description', arg_type=ts_description_type)
         c.argument('version_description', arg_type=ts_version_description_type)
+        c.argument('tag', tag_type)
+        c.argument('tags', tags_type)
```

Comment:
what is the difference for tag and tags?

Reply:
Customer can use tag for a single tag and tags for multiple.

Reply:
could we consider that use --tags for both single and multiple tags? I mean if there is one tag, use `--tags key=value1`, which would be more consistent with other commands. `--tag` and `--tags` would make cx confused

Reply:
@Juliehzl the customer can use either one. --tags works for a single tag as well. --tags and --tag map to the same input.  I expanded functionality for --tag since the code for this functionality is already available in the code based and used for other modules (try: az network vnet create --help)

Reply:
it seems there is only `--tags` in `az network vnet create`
![image](https://user-images.githubusercontent.com/49508232/102035795-4db9b080-3dfc-11eb-9130-1c25c65e56e3.png)

Reply:
as you know, `--tags` could support one or multiple tags, so I think it is unnecessary to add `--tag` and it would align with other commands

Reply:
@Juliehzl  Done.